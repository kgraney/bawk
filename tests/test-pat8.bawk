
/4e47 value:int1/ {
	print(value);
	print(RP);
	/value value2:int2/ {
		print(value2);
		print(RP);
	}
}

