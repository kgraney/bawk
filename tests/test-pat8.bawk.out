13
5
18760
14
